src/abi_compat.c

```diff
@@ -179,6 +179,7 @@ _write_netplan_conf(const char* netdef_id, const char* rootdir)
         g_warning("_write_netplan_conf: netdef_id (%s) not found.", netdef_id); // LCOV_EXCL_LINE
 }
 
+// LCOV_EXCL_START
```

Comment:
This is missing a `LCOV_EXCL_STOP` clause (or the middle `LCOV_EXCL_START` clause is superfluous).

Reply:
Removed the middle `LCOV_EXCL_START`.